troubleshooting.md

```diff
@@ -173,3 +173,21 @@ cat ~/.config/containers/storage.conf
   [storage.options]
     mount_program = "/bin/fuse-overlayfs"
 ```
+
+### 8) Permission denied when running systemd within a Podman container
+
+If you want to run systemd as PID 1 inside of a container on an SELinux
+separated machine, systemd needs to write to the cgroup file system.
+
+#### Symptom
+
+Systemd getse permission denied when attempting to write to the cgroup file
+system, and AVC messages start to show up in the audit.log file or journal on
+your system.
+
+#### Solution
+
+SELinux provides a Boolean, `container_manage_cgroup`,  that allows container
+processes to write to the cgroup file system. You need to turn on this boolean, on SELinux separated systems.
+
+`setsebool -P container_manage_cgroup 1`
```

Comment:
I find it's strange that this is a bool that gets set to 1 and not true.  But shrug.

Reply:
True will work. 

These all work
```
sh-4.4# setsebool -P container_manage_cgroup true
sh-4.4# setsebool -P container_manage_cgroup on
sh-4.4# setsebool -P container_manage_cgroup 1
```
Just typing 1 is the easiest.  :^)